cucumber/authenticators_gce/features/authn_gce.feature

```diff
@@ -33,12 +33,41 @@ Feature: GCE Authenticator - Hosts can authenticate with GCE authenticator
     And I set "authn-gce/service-account-email" annotation to host "test-app"
     And I set "authn-gce/project-id" annotation to host "test-app"
     And I set "authn-gce/instance-name" annotation to host "test-app"
-    And I obtain a GCE identity token in "full" format with audience claim value: "conjur/cucumber/host/test-app"
+    And I obtain a GCE identity token in full format with audience claim value: "conjur/cucumber/host/test-app"
```

Comment:
why did you remove the quotes around `full`?

Reply:
because it possible and much cleaner.